### DISCUSSION TITLE
Is it possible, to input a "Survey Variables" into the "CLI args"?

### DISCUSSION THREAD

**Discussion by kyberorg** [semaphoreui/semaphore#1352] [Q&A]
Use case: 
I have: 
* Survey Variables:  `host` (type `string`).
 
I want use it to limit playbook with single host or comma-separated list of hosts:
```
[
  "-l", "$host"
]
```

Is it possible?

**Comment by ansibleguy** [comment]
Greetings.

It seems this [question was already asked](https://github.com/ansible-semaphore/semaphore/issues/114#issuecomment-1335793364) but stayed unanswered.. :(
Actually I don't think so - as those survey-vars seem to be translated into 'extra-vars' that cannot be referenced in the context of the command-execution. I read into the codebase a little, but did not manage to find out how they are handled.
But please just try it out and report back. Would be interesting.

In practice '--limit' is used so common that, in my optinion, it deserves its own form-field inside the task & task-template.

\- AnsibleGuy

**Reply by kyberorg** [reply]
I agree, that limit should have own form-field.

**Comment by ansibleguy** [comment]
Opened a specific feature-request for adding a `--limit` field: https://github.com/ansible-semaphore/semaphore/issues/1357
